scheduler/generic_sched_test.go

```diff
@@ -604,34 +604,117 @@ func TestServiceSched_JobRegister_DistinctProperty_TaskGroup_Incr(t *testing.T)
 
 // Test job registration with spread configured
 func TestServiceSched_Spread(t *testing.T) {
-	h := NewHarness(t)
 	assert := assert.New(t)
 
-	// Create a job that uses spread over data center
+	start := uint32(100)
+	step := uint32(10)
+
+	for i := 0; i < 10; i++ {
+		name := fmt.Sprintf("%d%% in dc1", start)
```

Comment:
this test cycles through combinations of spread across both data centers, going from (100, 0) to (0, 100) increments of 10% at a time.